### PR TITLE
22029 get rid of default scope in content node

### PR DESCRIPTION
**Pull request by v-tsvid**

### CODE REVIEW

File: app/models/cms/content_node.rb [hunk 1]
```diff
@@ -169,23 +160,8 @@ def correct_redirect
       end
     end
 
-    def redirect_path
```
Comment (larskuhnt) [inline app/models/cms/content_node.rb:172]: was this  not used or why did you delete it?

**Review by larskuhnt — APPROVED** [review]

File: app/models/cms/content_node.rb (hunk 1 above)
Comment (v-tsvid) [reply app/models/cms/content_node.rb:172]: yeah, I couldn't find a place where this is used and couldn't find any specs that would fail because of it. Probably a dead code
